components/general/AddressItem.tsx

```diff
@@ -0,0 +1,31 @@
+import styles from "@/styles/AddressItem.module.css";
+import { fontSatoshi, fontRobotoMono, fontSyne } from "@/pages/_app";
+
+interface AddressItemProps {
+  imgSrc: string;
+  title: string;
+  street: string;
+  city: string;
+}
+
+const AddressItem = ({ imgSrc, title, street, city }: AddressItemProps) => {
+  return (
+    <div className={styles.address}>
+      <img src={imgSrc} className={styles.address__img} />
```

Comment:
It seems that you ignored the border radius
<img width="665" alt="image" src="https://user-images.githubusercontent.com/70834897/222850621-9f523afb-47fe-4068-b549-a55f557a26dd.png">

Reply:
Quite right.